apps/antalmanac/tests/buildings-type.test.ts

```diff
@@ -0,0 +1,18 @@
+import { describe, test, expect } from 'vitest';
+import { RepeatingCustomEvent, RepeatingCustomEventSchema } from '@packages/antalmanac-types';
+
+describe('building type', () => {
```

Comment:
Sorry I didn't specify, but can we make this file `tests/custom-events.test.ts`? And the description for the test suite would be `CustomEvents` or something similar.

```suggestion
describe('CustomEvents', () => {
```

Reply:
Gotcha, will do